packages/carbon-graphs/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
   * Added allowCalibration support for the x-axis on a line graph.
   * Finished implementation for `config.axis.x.allowCalibration` for a paired result graph with a numerical and timeseries x-axis.
   * Finished implementation for `config.axis.x.allowCalibration` for a scatter graph with a numerical and timeseries x-axis.
+  * Finished implementation for `config.axis.x.allowCalibration` for a bar graph with a numerical and timeseries x-axis.
```

Comment:
I would update this to match: https://github.com/cerner/terra-graphs/blob/0db73f925acd20425bf93ab5f49c1b162126126d/packages/carbon-graphs/CHANGELOG.md?plain=1#L6-L8
```suggestion
  * Added support for `config.axis.x.allowCalibration` for a bar graph with a numerical and timeseries x-axis.
```

Reply:
Fixed [149ec3e](https://github.com/cerner/terra-graphs/pull/293/commits/149ec3e9c1704f514ee29f51618f5840f4d650fb)